pkg/channel/grpc/grpc_channel.go

```diff
@@ -69,6 +73,10 @@ func (g *Channel) GetAppConfig() (*config.ApplicationConfig, error) {
 
 // InvokeMethod invokes user code via gRPC.
 func (g *Channel) InvokeMethod(ctx context.Context, req *invokev1.InvokeMethodRequest) (*invokev1.InvokeMethodResponse, error) {
+	if g.appHealth != nil && g.appHealth.GetStatus() != apphealth.AppStatusHealthy {
```

Comment:
Can we default the app channel to always be healthy instead? This way we don't need to check if appHealth is not nil.

Reply:
The nil check is to avoid instantiating the `appHealth` object if not needed.